Brick.ttl file is missing causing unit test failure

```================================================== short test summary info =================================================== 
ERROR tests/test_class_structure.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick.ttl' 
ERROR tests/test_definitions.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick.ttl'     
ERROR tests/test_hierarchy_inference.py
ERROR tests/test_inference.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick.ttl'       
ERROR tests/test_no_inference.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick.ttl'    
ERROR tests/test_quantities.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick.ttl'      
ERROR tests/test_shapes.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick.ttl'
ERROR tests/test_subclass_hierarchy.py - FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\Brick\\Brick...!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 8 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=============================================== 3 warnings, 8 errors in 47.26s =============================================== 
```
System: Windows 10   Python 3.10.7

Since the test are passing in the build, I'm assuming this is operator error.  Happy to update the docs if someone can tell me what I'm doing wrong.
Have a couple things I'd like to update in the docs anyway..

I tried using Make based on the help in 'CONTRIBUTING.md':

```
$ make test
mkdir -p extensions
python tools/sort_definitions.py bricksrc/definitions.csv
Traceback (most recent call last):
  File "C:\Projects\Brick\Brick\tools\sort_definitions.py", line 10, in <module>
    data = list(reader)
  File "C:\Users\FinchMik\AppData\Local\Programs\Python\Python310\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 5393: character maps to <undefined>
make: *** [Makefile:5: Brick.ttl] Error 1
```
Then I tries using the help from 'README.md'
```Run tests by executing `pytest` in the top-level directory of this repository.```

I tried running `make test` again and it generated a bunch of this error, 
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Projects\Brick\Brick\.venv\lib\site-packages\rdflib\term.py", line 2084, in _castLexicalToPython    
    return conv_func(lexical)  # type: ignore[arg-type]
  File "C:\Projects\Brick\Brick\.venv\lib\site-packages\rdflib\term.py", line 1652, in _parseHTML
    raise ImportError(
ImportError: HTML5 parser not available. Try installing html5lib <http://code.google.com/p/html5lib>
Failed to convert Literal lexical form to value. Datatype=http://www.w3.org/1999/02/22-rdf-syntax-ns#HTML, Converter=<function _parseHTML at 0x000001E032571630>
Traceback (most recent call last):
  File "C:\Projects\Brick\Brick\.venv\lib\site-packages\rdflib\term.py", line 1645, in _parseHTML
    import html5lib
ModuleNotFoundError: No module named 'html5lib'
```
Tests then started running
========================================== short test summary info =========================================== 
FAILED tests/test_entity_properties.py::test_entity_property_type_inference - FileNotFoundError: [Errno 2] N...FAILED tests/test_inference.py::test_shacl_owl_equivalentclass - FileNotFoundError: [Errno 2] No such file o...====================== 2 failed, 53 passed, 1 skipped, 27 warnings in 781.39s (0:13:01) ====================== 
make: *** [Makefile:23: test] Error 1

With the patch:
`py generate_brick.py` does not generate any errors except for not having docker. Just a lot of warnings.
`pytest` has errors:
```=============================================== short test summary info ================================================ 
FAILED tests/test_entity_properties.py::test_entity_property_type_inference - FileNotFoundError: [Errno 2] No such fil...FAILED tests/test_inference.py::test_shacl_owl_equivalentclass - FileNotFoundError: [Errno 2] No such file or director...========================== 2 failed, 53 passed, 1 skipped, 13 warnings in 1270.61s (0:21:10) =========================== ```

Make still generates an error:
```$ make test
mkdir -p extensions
python tools/sort_definitions.py bricksrc/definitions.csv
python generate_brick.py
C:\Projects\Brick\Brick\.venv\lib\site-packages\brickschema\graph.py:240: UserWarning: Could not load Reasonable reasoner. Needs 'reasonable' option during install.
  warn(
2022-10-11:17:46:01,387 ERROR   [inference.py:98] Could not connect to docker (Error while fetching server API version: (2, 'CreateFile', 'The system cannot find the file specified.')); defaulting to naive evaluation
C:\Projects\Brick\Brick\.venv\lib\site-packages\brickschema\graph.py:250: UserWarning: Could not load Allegro reasoner. Needs 'allegro' option during install.
  warn(
C:\Projects\Brick\Brick\.venv\lib\site-packages\brickschema\graph.py:240: UserWarning: Could not load Reasonable reasoner. Needs 'reasonable' option during install.
  warn(
2022-10-11:17:46:20,399 ERROR   [inference.py:98] Could not connect to docker (Error while fetching server API version: (2, 'CreateFile', 'The system cannot find the file specified.')); defaulting to naive evaluation
2022-10-11:17:46:28,588 WARNING [generate_brick.py:297] Property 'tags' not defined for https://brickschema.org/schema/Brick#Measurable
2022-10-11:17:46:28,588 WARNING [generate_brick.py:297] Property 'tags' not defined for https://brickschema.org/schema/Brick#Measurable
2022-10-11:17:46:29,662 WARNING [generate_brick.py:297] Property 'tags' not defined for https://brickschema.org/schema/Brick#Lighting
2022-10-11:17:46:29,683 WARNING [generate_brick.py:297] Property 'tags' not defined for https://brickschema.org/schema/Brick#PV_Generation_System
Traceback (most recent call last):
  File "C:\Projects\Brick\Brick\generate_brick.py", line 880, in <module>
    add_definitions()
  File "C:\Projects\Brick\Brick\generate_brick.py", line 644, in add_definitions
    term = URIRef(definition[0])
IndexError: list index out of range
make: *** [Makefile:6: Brick.ttl] Error 1
(.venv)```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -204,7 +204,8 @@ To set up the development environment
     . brickvenv/bin/activate
     pip install -r requirements.txt
     ```
-    You do not need to install the `docker` dependency if your system does not support it. Use of docker enables use of Allegrograph, which can speed up some of the Brick compilation processes.
+    * You do not need to install the `docker` dependency if your system does not support it. Use of docker enables use of Allegrograph, which can speed up some of the Brick compilation processes.
+    * If C++ compiler is not installed you may get an error installing the Levenshtein-search.  It is not strickly necessary to build and test brick.  You can safely ignore this error.
 4. Make the desired changes **to the Python files**. Do not edit the `Brick.ttl` or `Brick+extensions.ttl` files directly.
 5. Compile the `Brick.ttl` and `Brick_expanded.ttl` files by executing the `generate_brick.py` script using Python
     ```bash

--- a/README.md
+++ b/README.md
@@ -52,7 +52,7 @@ Tests go in the `tests/` directory and should be implemented using [pytest](http
 [`tests/test_inference.py`](https://github.com/BrickSchema/Brick/blob/master/tests/test_inference.py) is a good example.
 
 Run tests by executing `pytest` or `make test` in the top-level directory of this repository.
-* Before running `pytest` the Brick.ttl file needs to be created using either `make` or `py generate_brick.py`. 
+* Before running `pytest` the Brick.ttl file needs to be created using either `make` or `python generate_brick.py`. 
 
 ## Python Framework
 

--- a/README.md
+++ b/README.md
@@ -16,13 +16,13 @@ This repository tracks the main schema development of Brick.
 
 ## Discussion
 
-Discussion takes place primarily on the Brick User Form: [https://groups.google.com/forum/#!forum/brickschema](https://groups.google.com/forum/#!forum/brickschema)
+Discussion takes place primarily on the Brick User Forum: [https://groups.google.com/forum/#!forum/brickschema](https://groups.google.com/forum/#!forum/brickschema)
 
 ## Questions and Issues
 
 If you have an issue with Brick's coverage, utility or usability, or any other Brick-related question:
 
-1. First check the [Brick user form](https://groups.google.com/forum/#!forum/brickschema) and the [Brick issue tracker](https://github.com/BuildSysUniformMetadata/Brick/issues)
+1. First check the [Brick user forum](https://groups.google.com/forum/#!forum/brickschema) and the [Brick issue tracker](https://github.com/BuildSysUniformMetadata/Brick/issues)
    to check if anyone has asked your question already.
 2. If you find a previously submitted issue that closely mirrors your own, feel free to jump in on the conversation. Otherwise, please file a new issue or submit a new thread on the forum.
 
@@ -51,7 +51,8 @@ See [CONTRIBUTING.md](https://github.com/BrickSchema/Brick/blob/master/CONTRIBUT
 Tests go in the `tests/` directory and should be implemented using [pytest](https://pytest.readthedocs.io/en/latest/getting-started.html#getstarted).
 [`tests/test_inference.py`](https://github.com/BrickSchema/Brick/blob/master/tests/test_inference.py) is a good example.
 
-Run tests by executing `pytest` in the top-level directory of this repository.
+Run tests by executing `pytest` or `make test` in the top-level directory of this repository.
+* Before running `pytest` the Brick.ttl file needs to be created using either `make` or `py generate_brick.py`. 
 
 ## Python Framework
 

--- a/tests/test_entity_properties.py
+++ b/tests/test_entity_properties.py
@@ -1,6 +1,7 @@
 from rdflib import Namespace, Literal
 from brickschema.namespaces import BRICK, A, REF, XSD
 import brickschema
+import os
 
 
 def test_entity_property_validation():
@@ -60,13 +61,13 @@ def test_entity_property_type_inference():
     valid, _, report = g.validate()
     assert valid, report
     g.expand("shacl")
-    g.serialize("/tmp/test.ttl", format="ttl")
+    g.serialize("test.ttl", format="ttl")
 
     res = g.query(
         "SELECT ?ref WHERE { ?point ref:hasExternalReference ?ref . ?ref a ref:BACnetReference }"
     )
     assert len(res) == 1
-
+    os.remove("test.ttl")
 
 def test_last_known_value():
     g = brickschema.Graph()

--- a/tests/test_inference.py
+++ b/tests/test_inference.py
@@ -2,7 +2,7 @@ from rdflib import RDF, RDFS, OWL, Namespace, Literal
 import pytest
 import brickschema
 from .util import make_readable
-import sys
+import os, sys
 
 sys.path.append("..")
 from bricksrc.namespaces import BRICK, TAG, A, SKOS  # noqa: E402
@@ -191,14 +191,16 @@ def test_shacl_owl_equivalentclass():
     g.load_file("Brick.ttl")
     g.add((BLDG.VAV1, A, BRICK.VAV))
     g.add((BLDG.VAV2, A, BRICK.Variable_Air_Volume_Box))
-    g.serialize("/tmp/x.ttl", format="turtle")
+    g.serialize("x.ttl", format="turtle")
     g.expand("shacl")
-    g.serialize("/tmp/y.ttl", format="turtle")
+    g.serialize("y.ttl", format="turtle")
     res = g.query("SELECT ?vav WHERE { ?vav rdf:type brick:VAV }")
     assert len(list(res)) == 2, "VAV should be equivalent to Variable_Air_Volume_Box"
 
     res = g.query("SELECT ?vav WHERE { ?vav rdf:type brick:Variable_Air_Volume_Box }")
     assert len(list(res)) == 2, "VAV should be equivalent to Variable_Air_Volume_Box"
+    os.remove("x.ttl")
+    os.remove("y.ttl")
 
 
 def test_meter_inference():

--- a/tools/sort_definitions.py
+++ b/tools/sort_definitions.py
@@ -5,7 +5,7 @@ if len(sys.argv) < 2:
     print("Usage: python sort_definitions.py <input_file>")
     sys.exit(1)
 
-with open(sys.argv[1], "r") as f:
+with open(sys.argv[1], "r", encoding="utf-8") as f:
     reader = csv.reader(f)
     data = list(reader)
     # remove duplicates
@@ -20,6 +20,6 @@ with open(sys.argv[1], "r") as f:
     # sort by term
     terms = sorted(terms, key=lambda tup: tup[0])
 
-with open(sys.argv[1], "w") as f:
+with open(sys.argv[1], "w", encoding="utf-8") as f:
     writer = csv.writer(f)
     writer.writerows(terms)

